src/main/java/it/pagopa/transactions/commands/handlers/TransactionSendClosureHandler.java

```diff
@@ -141,37 +139,33 @@ public Mono<Either<TransactionClosureErrorEvent, TransactionEvent<TransactionClo
-                                            authRequestData.authorizationCode(),
-                                            "rrn",
-                                            ECOMMERCE_RRN,
-                                            "fee",
-                                            fee.toString(),
-                                            "timestampOperation",
-                                            // FIXME Pass the timestamp of the authorization result
-                                            // bug CHK-1410: date formatted to yyyy-MM-ddTHH:mm:ss truncating millis
-                                            OffsetDateTime.now()
-                                                    .toLocalDateTime()
-                                                    .truncatedTo(ChronoUnit.SECONDS)
-                                                    .format(DateTimeFormatter.ISO_LOCAL_DATE_TIME),
-                                            "totalAmount",
-                                            totalAmount.toString()
-                                    )
-                            );
-
+                            .transactionId(tx.getTransactionId().value());
+
+                    if (ClosePaymentRequestV2Dto.OutcomeEnum.OK.equals(closePaymentRequest.getOutcome())) {
+                        closePaymentRequest.idPSP(transactionAuthorizationRequestData.getPspId())
+                                .idBrokerPSP(transactionAuthorizationRequestData.getBrokerName())
+                                .idChannel(transactionAuthorizationRequestData.getPspChannelCode())
+                                .totalAmount(totalAmount)
+                                .fee(fee)
+                                .timestampOperation(updateAuthorizationRequestDto.getTimestampOperation())
+                                .paymentMethod(transactionAuthorizationRequestData.getPaymentTypeCode())
+                                .additionalPaymentInformations(
+                                        new AdditionalPaymentInformationsDto()
+                                                .tipoVersamento(TIPO_VERSAMENTO_CP)
+                                                .outcomePaymentGateway(
+                                                        AdditionalPaymentInformationsDto.OutcomePaymentGatewayEnum
+                                                                .fromValue(authRequestData.outcome())
+                                                )
+                                                .authorizationCode(authRequestData.authorizationCode())
+                                                .fee(fee)
+                                                .timestampOperation(
+                                                        updateAuthorizationRequestDto.getTimestampOperation()
+                                                                .toZonedDateTime().truncatedTo(ChronoUnit.SECONDS)
```

Comment:
```suggestion
                                                        updateAuthorizationRequestDto.getTimestampOperation()
```
Remove truncation. Before this api modification this field was a string and was required to be formatted as a date truncating millis, now is an offset datetime so it can be just valued with the updateAuthorizationRequestDto.getTimestampOperation field value